### ISSUE TITLE
Vite server indefinitely hanging

### ISSUE THREAD

**Issue by KilDesu** [quaffui/quaff#81]
Not sure if it's only on my side but when vite server is reloading after saving a file while on the "components/qtabs" page, the page reloads indefinitely without any error or warning.

I think it's since my refactor of QTabs, but I don't see what part of my code could induce this bug...

Can you test this @therealcoder1337? Do you have any idea what could cause this?

**Comment by therealcoder1337** [comment]
@KilDesu Could you please test with the `update-deps` branch? https://github.com/quaffui/quaff/tree/update-deps
You have to use `npm i --force` currently. Don't forget to use `npm i` again if you change back to another branch later.
With that branch I tried changing `src/lib/components/tabs/QTabs.svelte` while on its docs page and didn't notice any issues.

**Comment by KilDesu** [comment]
@therealcoder1337 Still doesn't work, even on this branch.
Here is exactly what I do to reproduce it:
- `pnpm dev`: start the dev server
- Go to `localhost:5173/components/tabs` page
- Change the tabs' value on the page by clicking on some of them
- Go back to VSCode and save `QTabs.svelte`: vite server restarts normally
- Go back to firefox on the tabs doc page and press `CTRL+R` to reload the page
=> The page reloads indefinitely, I can't make it reload without stopping the dev server and restarting it.

This bug seems to happen only on the tabs doc page.

NB: While I tested this again after writing this message, the bug didn't seem to happen anymore (even though it happened just before I wrote this message)...
Well, closing this issue for now, but it's really weird...
